javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,98 @@
+let userName = null; 
+let todoList = []; 
+
+function getReply(command) {
+  const lowerCaseCommand = command.toLowerCase();
```

Comment:
Good thinking! It's always nice to make an effort to handle slightly incorrect user input, such as casing.